eldritch/test_gates.py

```diff
@@ -5,16 +5,19 @@
 import sys
 import unittest
 from unittest import mock
+from typing import cast
 
 # Hack to allow the test to be run directly instead of invoking python from the base dir.
+import eldritch.places
```

Comment:
Why did this move? And also get imported as eldritch.places instead of just places? This should probably get imported below the hack that changes the import paths to make this discoverable.